commodore/package/compile.py

```diff
@@ -20,15 +20,9 @@
 from commodore.postprocess import postprocess_components
 
 
-# pylint: disable=too-many-arguments disable=too-many-locals
-def compile_package(
-    cfg: Config,
-    pkg_path_: str,
-    root_class: str,
-    value_files_: Iterable[str],
-    tmp_dir: Optional[str] = "",
-    keep_dir: bool = False,
-):
+def set_work_dir(
```

Comment:
```suggestion
def create_and_set_work_dir(
```

Would be a bit more expressive.